scanner/src/main/kotlin/Scanner.kt

```diff
@@ -193,7 +193,7 @@ class Scanner(
 
         val vcsPathsForProvenances = getVcsPathsForProvenances(provenances)
 
-        val filteredScanResults = filterScanResultsByVcsPaths(controller.getAllScanResults(), vcsPathsForProvenances)
+        val scanResults = filterScanResultsByVcsPaths(controller.getAllScanResults(), vcsPathsForProvenances)
```

Comment:
commit-msg: 

Nit: We normally start with capitalized after the `:`, e.g. "**E**xtract".